### PR TITLE
[de] Update Exclusion list

### PR DESCRIPTION
**Pull request by sotten**
add `minute` and `name` to Exclusion list avoid missing translation in `validation-attributes.php`.

### CODE REVIEW

**Comment by andrey-helldar** [comment]
@caouecs, wait, don't take PR. I need to check.

@sotten, the exception file should contain the keys, but if I specify `name` or `attributes.name`, the `php app/excludes.php` script removes those values.

I will fix it shortly.

File: excludes/de.php [hunk 1]
```diff
@@ -82,11 +82,13 @@
     'Martinique',
     'Mauritius',
     'Mayotte',
+    'minute',
```
Comment (andrey-helldar) [inline excludes/de.php:85]: Replace with `Minute` (value)

File: excludes/de.php [hunk 2]
```diff
@@ -82,11 +82,13 @@
...
     'Monaco',
     'Montenegro',
     'Montserrat',
     'Myanmar',
     'Name',
+    'name',
```
Comment (andrey-helldar) [inline excludes/de.php:91]: Remove this line (`Name` already defined).

**Review by andrey-helldar — CHANGES_REQUESTED** [review]
@caouecs, PR https://github.com/Laravel-Lang/lang/pull/1867 for fix comparing already pushed.

**Review by andrey-helldar — APPROVED** [review]

**Comment by andrey-helldar** [comment]
@caouecs, it's ok.

**Comment by caouecs** [comment]
Danke

**Pull request merged**